spec/lib/french_toast/notifier_spec.rb

```diff
@@ -30,5 +30,50 @@
       expect(ActionCable.server).
         to have_received(:broadcast).with(session_key, payload)
     end
+
+    context "when a user is connected via ActionCable" do
+      it "deletes the session's LastNotification" do
+        session_key = "a user's session key"
+        payload = "a payload to send"
+        fake_redis = instance_double(Redis)
+        allow(Redis).to receive(:new).and_return(fake_redis)
+        allow(fake_redis).to receive(:pubsub).
+          with("channels", "action_cable/*").
+          and_return([
+            "action_cable/#{session_key}",
+            "action_cable/asdfasdfasdgasdga",
+          ])
+        allow(ApplicationController).to receive(:render).and_return(payload)
+        allow(ActionCable.server).to receive(:broadcast)
+
+        described_class.new(session_key).
+          notify(:partial_name, locals: { foo: :bar })
+
+        expect(FrenchToast::LastNotification.where(session: session_key)).
+          to be_empty
+      end
+    end
+
+    context "when users is not connected via ActionCable" do
+      it "does not delete the session's LastNotification" do
+        session_key = "a user's session key"
+        payload = "a payload to send"
+        fake_redis = instance_double(Redis)
+        allow(Redis).to receive(:new).and_return(fake_redis)
+        allow(fake_redis).to receive(:pubsub).
+          with("channels", "action_cable/*").
+          and_return([
+            "action_cable/asdfasdfasdgasdga",
```

Comment:
These two tests have almost identical setup, which made it tricky to figure out what was different. Can a method or two be extracted around this? At least something like:

```ruby
subscribe_to_channels(["action_cable/asdfasdfasdgasdga"])
```